File: 프로그래머스/lv0/120835. 진료 순서 정하기/진료 순서 정하기.cs
using System;
using System.Linq;

public class Solution {
    public int[] solution(int[] emergency) {
        int[] sortedArray = emergency.OrderByDescending(n => n).ToArray();
        return emergency.Select(n => Array.IndexOf(sortedArray, n) + 1).ToArray();
    }
}

File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> emergency) {
    vector<int> answer = vector<int>(emergency.size(), 1);
    
    for (int i = 0; i < emergency.size(); ++i)
    {
        for (int k : emergency)
        {
            if (emergency[i] < k)
                answer[i]++;
        }
    }
    
    return answer;
}